extensions/amp-story/1.0/amp-story-share.js

```diff
@@ -277,13 +320,35 @@ export class ShareWidget {
     // TODO(alanorozco): Listen for proper tap event (i.e. fastclick)
     listen(linkShareButton, 'click', e => {
       e.preventDefault();
-      this.copyUrlToClipboard_();
+      this.copyUrlToClipboard_(this.root.querySelector('#page-share').checked);
     });
   }
 
-  /** @private */
-  copyUrlToClipboard_() {
-    const url = Services.documentInfoForDoc(this.getAmpDoc_()).canonicalUrl;
+  /** On desktop mode, with branching, users should be able to share a story
+   * starting from a specific page.
+   * @private */
+  maybeAddPageShareButton_() {
+    if (isExperimentOn(this.win, 'amp-story-branching') && this.isDesktopUi_) {
+
+      const sharePageCheck =
+        renderAsElement(this.win.document, SHARE_PAGE_TEMPLATE);
+      sharePageCheck.querySelector(
+          '#page-share-span').innerHTML = 'Share this page';
```

Comment:
I think this should be rendered with the simple templates stuff?  We'll also want to localize it, so you can follow the instructions [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-story/1.0/_locales/README.md#adding-new-strings-english), and pass your `LocalizedStringId` to the simple templates def, like we do [here](https://github.com/ampproject/amphtml/blob/f603dafba0bb3bf0ecb788a919d37699162014f7/extensions/amp-story/1.0/amp-story-unsupported-browser-layer.js#L46-L47)

Reply:
Done